UserInterface/FileBrowse/source/PlayState.hx

```diff
@@ -17,6 +17,11 @@ import flash.display.Loader;
 import flash.display.LoaderInfo;
 import flash.net.FileReference;
 import flash.net.FileFilter;
+#elseif html5
```

Comment:
this should just be `#if (flash || html5)` openfl and flash imports are interchangeable in openfl/flixel

see here:
https://github.com/openfl/openfl/blob/develop/src/openfl/net/FileReference.hx#L1433
and here:
https://github.com/openfl/openfl/blob/develop/lib/flash-externs/src/flash/net/FileReference.hx#L31

Reply:
Thanks!